app/styles/components/c-other-section.scss

```diff
@@ -18,13 +18,36 @@
   }
 
   .other-logos {
-    @media #{$mq-tablet} {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    @media #{$mq-mobile-large} {
+      flex-direction: row;
       padding-left: 90px;
       padding-right: 90px;
     }
 
     img {
+      margin-bottom: 20px;
```

Comment:
Remove this margin in the media query because on desktop, the logo are not vertically centred.